options/rtld/generic/main.cpp

```diff
@@ -553,7 +553,16 @@ const char *__dlapi_error() {
 
 extern "C" [[ gnu::visibility("default") ]]
 void *__dlapi_get_tls(struct __abi_tls_entry *entry) {
+#if defined(__m68k__) && MLIBC_STATIC_BUILD
+	// Static binaries sometimes call this on m68k, and being static, there are
+	// no relocations so the object field doesn't get rewritten to point to the
+	// corresponding SharedObject. But since it's guaranteed to have the id of
+	// 0x1, this suffices.
+	__ensure((uintptr_t)entry->object == 0x1);
+	return reinterpret_cast<char *>(accessDtv(executableSO)) + entry->offset;
+#else
```

Comment:
Can this logic be moved into lsb/generic/tls.cpp, alongside an existing check for RISC-V? https://github.com/managarm/mlibc/blob/13a686b37da12d70dee27b9c6ffab4207feb4156/options/lsb/generic/tls.cpp#L5-L14

Reply:
Thanks, have done so.